guides/common/assembly_managing-suse-content.adoc

```diff
@@ -8,8 +8,12 @@ include::modules/proc_installing-the-scc-manager.adoc[leveloffset=+1]
 
 include::modules/proc_adding-an-scc-account.adoc[leveloffset=+1]
 
+include::modules/proc_removing-an-scc-account.adoc[leveloffset=+1]
```

Comment:
Can we keep `removing` at last, after the update?

Reply:
Applied.